.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_style = space
```

Comment:
I prefer using tabs. Maybe we shouldn't include this file in this PR.

Reply:
My bad, forgot to update this from the stuff I copied over. I can switch to tabs, but I'd really recommend having basic `.editorconfig` stuff in here, I'd recommend any repo have a minimal one to keep tab usage or whatever consistent, rather than having editors guess it.

We need it anyway for enforcing the linter globalization rules anyway, if you want I can remove the top stuff but I'd recommend keeping it.